analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectFunctionalTest.java

```diff
@@ -101,6 +101,37 @@ public void ensureAlignmentFileCreated() throws IOException, URISyntaxException,
                 .filteredOn(l -> l.trim().equals(AlignmentTask.APPLY_GME_REPOS)).hasSize(1);
     }
 
+    @Test
+    public void ensureAlignmentProduceCustomManipulationPlugin() throws IOException, URISyntaxException, ManipulationException {
+        final File projectRoot = tempDir.newFolder("simple-project");
+
+        final TestManipulationModel alignmentModel = TestUtils.align(
+                projectRoot, projectRoot.getName(),
+                Collections.singletonMap("manipulationVersion", "0.1.2.3.4"));
+
+        assertThat(alignmentModel).isNotNull().satisfies(am -> {
+            assertThat(am.getOriginalVersion()).isEqualTo("1.0.1");
+            assertThat(am.getGroup()).isEqualTo("org.acme.gradle");
+            assertThat(am.getName()).isEqualTo("root");
+        });
+        File gme = new File(projectRoot, AlignmentTask.GME);
+        assertTrue(gme.exists());
+        assertThat(FileUtils.readFileToString(gme, Charset.defaultCharset()))
+                .contains("org.jboss.gm:manipulation:0.1.2.3.4");
+        assertTrue(new File(projectRoot, AlignmentTask.GME).exists());
```

Comment:
Line 130 has assertThat().exists() but there has assertTrue().

Reply:
Do you mean for consistency or some other reason? I don't see why `assertTrue` shouldn't work?

Reply:
Yes, just for consistency of the calls within the same method and also that `assertThat()` will produce a more informative error message should the test ever fail.

Reply:
Actually there is a lot of usage of e.g. `assertTrue(new File(projectRoot, AlignmentTask.GME).exists());` in that file.

Reply:
Regex replace in intellij to the rescue.